assets/js/base/context/hooks/collections/use-collection.ts

```diff
@@ -44,7 +44,7 @@ export interface useCollectionOptions {
 	namespace: string;
 	resourceName: string;
 	resourceValues?: number[];
-	query: Query;
+	query: Record< string, unknown >;
```

Comment:
@alexflorisca I change this from `Query` to `Record< string, unknown >` because different endpoints accept different sets of queries.

Reply:
Ok, do you want me to review this PR? It's assigned to @tomasztunik at the moment

Reply:
Would that mean the Query in `woocommerce/types` is too specific or was it misused?